src/components/KonnectorSync.jsx

```diff
@@ -1,15 +1,26 @@
+import styles from '../styles/KonnectorSync'
+
 import React from 'react'
 import { translate } from 'cozy-ui/react/I18n'
 
 import DescriptionContent from './DescriptionContent'
 
-export const KonnectorSync = ({ t, f, date }) => {
+export const KonnectorSync = ({ t, f, date, error, editing, submitting, onForceConnection }) => {
   return (
     <div>
       { date && <DescriptionContent
         title={t('account.message.synced.title')}
         messages={[t('account.message.synced.last_sync', { date: f(date, t('account.message.synced.date_format')) })]}
       /> }
+      { !editing && !error && <div className={styles['account-forceConnection']}>
+        <button
+          className={submitting ? styles['submitting'] : ''}
```

Comment:
Two things here:
* @m4dz will confirm, but setting the `aria-busy` to `true` on any cozy button will make a spinner appear.
* This button should have its own state, and so `KonnectorSync` should become a Component. I don't think it's a good thing at UX level that when you click on a button, an other one got a spinner. Also, we need to keep the `AccountConnection` global `submitting` to disable the sync button.